content/docs/command-reference/import.md

```diff
@@ -228,3 +228,34 @@ outs:
 
 See a full explanation in our [Data Registries](/doc/use-cases/data-registries)
 use case.
+
+## Example: Importing from any Git repository
+
+You can even import files from plain Git repos that are not <abbr>DVC repositories</abbr>. 
+For example, let's import a
+dataset from [GSA's data repo](https://github.com/GSA/data). 
+
+```dvc
+$ dvc import git@github.com:GSA/data \
+           enterprise-architecture/it-standards.csv
+Importing ...
+```
+
+The file is imported, and along with it, an import stage
+([DVC-file](/doc/user-guide/dvc-file-format)) file is created. Check  
+`it-standards.csv.dvc`:
+
+```yaml
+deps:
+  - path: enterprise-architecture/it-standards.csv
+    repo:
+      url: git@github.com:GSA/data
+      rev_lock: af6a1feb542dc05b4d3e9c80deb50e6596876e5f
+outs:
+  - md5: 7e6de779a1ab286745c808f291d2d671
+    path: it-standards.csv
+```
+Notice in this snippet that the path and the url you specified earlier are saved
+in the `it-standards.csv.dvc` file, which means the imported file can later be
+updated using [`dvc update`](/doc/command-reference/update) from the upstream 
+repository.
```

Comment:
Oh actually I have another review comment:

This paragraph is good but not exactly what I was imagining. It's OK, you're only starting to learn DVC so let me help you here: The main things to notice from this YAML sample is that it saves the data source from where you imported, that is the subfields inside `repo:` If you find other `dvc import` examples throughout the docs, this is mentioned in a few places I think, so maybe you can mostly copy-paste from those other instances of similar examples. The note about update is not bad but probably not super relevant to this example so I would remove that part. Thanks